### ISSUE TITLE
New release on 3rd March breaks amp-validator

### ISSUE THREAD

**Issue by nataliecarey** [dorightdigital/amp-validator#4]
Every valid document is listing the same error:

```
amp-validator http://www.mirror.co.uk/3am/celebrity-news/suki-waterhouse-neelam-gill-set-7427293.amp

http://www.mirror.co.uk/3am/celebrity-news/suki-waterhouse-neelam-gill-set-7427293.amp failed validation
 -- Line 1:1750 The tag 'head
 -- Line 1:1792 The tag 'noscript
```

These errors are usually caused by `zombie` (the headless browser I've used) not supporting 'modern browser features' which AMP expects to exist.  These are monkeypatched here https://github.com/dorightdigital/amp-validator/blob/master/browser.js, presumably the new release uses a feature not yet supported.

**Comment by nataliecarey** [comment]
Resolved.  I hadn't previously treated errors and warnings uniquely, that release of the validator listed warnings which weren't reported in the browser.

This issue is resolved in `1.0.5` and on amp-validator.com
